api/envoy/config/core/v4alpha/config_source.proto

```diff
@@ -144,6 +144,14 @@ message RateLimitSettings {
   // Rate at which tokens will be filled per second. If not set, a default fill rate of 10 tokens
   // per second will be used.
   google.protobuf.DoubleValue fill_rate = 2 [(validate.rules).double = {gt: 0.0}];
+
+  // Initial retry delay in Milli Second. If not set, a
+  // default value of 500 will be used.
+  google.protobuf.UInt32Value retry_initial_delay_ms = 3;
```

Comment:
Instead of adding these two fields individually, it would probably make more sense to just use the existing [`BackoffStrategy` message](https://github.com/envoyproxy/envoy/blob/361b97ed5145063f040ca8ef6a05ae0521c4137f/api/envoy/config/core/v3/backoff.proto#L19).